source/doc-odm-user-guide/supported-formats.rst

```diff
@@ -31,7 +31,52 @@ Samples metadata is supplied in TSV format file. There needs to be a **Sample So
+- **Measurements (or values)**: These are the actual values captured for each feature under different conditions (e.g., gene expression values, protein abundance, pathway activity, sales volume, etc.).
+The first example below demonstrates the simplest and most popular type of data frame. Here, the features are genes listed in the first column, while the rest of the table contains measurements of gene expression for a number of samples. Each column represents a list of corresponding gene expression values in different samples (each column name represents a sample).
+
+.. image:: images/data-frame1.png
+   :scale: 40 %
+   :align: center
+
+In the second example, we present a more complex version of the data frame. Here, features are represented by more than one column containing additional information. Such columns might represent gene names, protein names, peptide sequences, PTM sites, pathway names, countries, etc. There can also be more than one type of measurements or calculated values for each feature in each sample. For example, for each gene (feature), we might have its expression level, quality flag, sequencing depth, Fold Change of differential expression, p-value, q-value per sample.
+
+.. image:: images/data-frame2.png
+   :scale: 40 %
+   :align: center
+
+This format provides a wide range of data types that can be uploaded and indexed in ODM.
+
+However, the current **BETA** version has some limitations on the file content. Please ensure your file adheres to the following requirements:
+
+- All columns must have names.
+- All feature columns must be consecutive on the left side of the file. You need to explicitly specify the number of feature columns during the uploading process.
+- Missing values: If your file intentionally contains missing data/values, ensure such values are coded as one of the following: " " (as a space symbol), empty (no symbol between two tabs), “NaN”, “null”, “N/A”, “N\A”, “NA”, “filtered”, “Inf”, “-Inf”.
+- The system automatically identifies all feature columns as either string value or numeric value columns. If a column that should be numeric contains at least one value with a non-numeric character (except for the missing value coded as indicated above), it will be considered a string value column, disabling the ranged search capabilities.
+- Columns with measurements must contain only either numeric values or missing values (as specified above).
+- If your file contains more than one measurement per Sample (Library or Preparation), e.g., Fold Change and P-value, the system will automatically recognize it using the following criteria:
+ - The column name contains a dot symbol as a separator between the Sample (Library or Preparation) name and the measurement type. If the column name contains more than one dot symbol (e.g., Sample1.p.value ), the first dot will be used as a separator.
+ - All columns must contain the dot symbol.
```

Comment:
valid for multimeasurement case only (not obvious from the context to me, as a reader of the document)